configs/default/HENA-TALONF7DJIHD.config

```diff
@@ -0,0 +1,146 @@
+# Betaflight / STM32F7X2 (S7X2) 
+
+board_name TALONF7DJIHD
+manufacturer_id HENA
+
+# resources
+resource BEEPER 1 B04
+resource MOTOR 1 B06
+resource MOTOR 2 B07
+resource MOTOR 3 B08
+resource MOTOR 4 B09
+resource MOTOR 5 A01
+resource MOTOR 6 C08
+resource MOTOR 7 C09
+resource MOTOR 8 B01
+resource LED_STRIP 1 B01
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 2 A02
+resource SERIAL_TX 3 B10
+resource SERIAL_TX 4 A00
+resource SERIAL_TX 5 C12
+resource SERIAL_TX 6 C06
+resource SERIAL_RX 1 A10
+resource SERIAL_RX 2 A03
+resource SERIAL_RX 3 B11
+resource SERIAL_RX 4 A01
+resource SERIAL_RX 5 D02
+resource SERIAL_RX 6 C07
+resource LED 1 B00
+resource RX_BIND 1 B02
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_SCK 3 C10
+resource SPI_MISO 1 A06
+resource SPI_MISO 2 B14
+resource SPI_MISO 3 C11
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 2 B15
+resource SPI_MOSI 3 B05
+resource CAMERA_CONTROL 1 B03
+resource ADC_BATT 1 C02
+resource ADC_RSSI 1 C03
+resource ADC_CURR 1 C01
+resource SDCARD_CS 1 B12
+resource SDCARD_DETECT 1 C03
+resource PINIO 1 A14
+resource FLASH_CS 1 B12
+resource OSD_CS 1 A15
+resource GYRO_EXTI 1 C04
+resource GYRO_EXTI 2 C14
+resource GYRO_CS 1 A04
+resource GYRO_CS 2 C13
+resource USB_DETECT 1 C05
+
+# timer
+timer A03 AF3
+# pin A03: TIM9 CH2 (AF3)
+timer B00 AF1
+# pin B00: TIM1 CH2N (AF1)
+timer B01 AF2
+# pin B01: TIM3 CH4 (AF2)
+timer A01 AF2
+# pin A01: TIM5 CH2 (AF2)
+timer A00 AF2
+# pin A00: TIM5 CH1 (AF2)
+timer C06 AF2
+# pin C06: TIM3 CH1 (AF2)
+timer C07 AF3
+# pin C07: TIM8 CH2 (AF3)
+timer B05 AF2
+# pin B05: TIM3 CH2 (AF2)
+timer B09 AF2
+# pin B09: TIM4 CH4 (AF2)
+timer B08 AF2
+# pin B08: TIM4 CH3 (AF2)
+timer B03 AF1
+# pin B03: TIM2 CH2 (AF1)
+timer B06 AF2
+# pin B06: TIM4 CH1 (AF2)
+timer B07 AF2
+# pin B07: TIM4 CH2 (AF2)
+timer C08 AF3
+# pin C08: TIM8 CH3 (AF3)
+timer C09 AF3
+# pin C09: TIM8 CH4 (AF3)
+
+# dma
+dma ADC 1 1
+# ADC 1: DMA2 Stream 4 Channel 0
+dma pin B00 0
+# pin B00: DMA2 Stream 6 Channel 0
+dma pin B01 0
+# pin B01: DMA1 Stream 2 Channel 5
+dma pin A01 0
+# pin A01: DMA1 Stream 4 Channel 6
+dma pin A00 0
+# pin A00: DMA1 Stream 2 Channel 6
+dma pin C06 0
+# pin C06: DMA1 Stream 4 Channel 5
+dma pin C07 0
+# pin C07: DMA2 Stream 2 Channel 0
+dma pin B05 0
+# pin B05: DMA1 Stream 5 Channel 5
+dma pin B08 0
+# pin B08: DMA1 Stream 7 Channel 2
+dma pin B03 0
+# pin B03: DMA1 Stream 6 Channel 3
+dma pin B06 0
+# pin B06: DMA1 Stream 0 Channel 2
+dma pin B07 0
+# pin B07: DMA1 Stream 3 Channel 2
+dma pin C08 0
+# pin C08: DMA2 Stream 2 Channel 0
+dma pin C09 0
+# pin C09: DMA2 Stream 7 Channel 7
+
+# feature
+feature -RX_PARALLEL_PWM
+feature RX_SERIAL
+feature OSD
+
+# serial
+serial 4 64 115200 57600 0 115200
+serial 5 1 115200 57600 0 115200
+
+# master
+set mag_hardware = NONE
+set baro_hardware = NONE
```

Comment:
If `mag_hardware = NONE` and `baro_hardware = NONE` is kept in here then Betaflight will not be supporting addition of baro or mag to this hardware, and users will be told to buy different hardware if they want to use baro or mag.
If you don't want this please remove these lines.